src/components/organizations/cluster_detail.js

```diff
@@ -212,6 +212,18 @@ class ClusterDetail extends React.Component {
       }
     }
 
+    var availabilityZonesOrNothing = null;
+    if (this.props.cluster.availability_zones) {
```

Comment:
Q: does the API omit availability zones if there are none? 
A: yes it does (https://github.com/giantswarm/api/blob/master/server/endpoint/cluster/searcher/response.go#L20)

So, thats ok.